CHANGES.rst

```diff
@@ -3,6 +3,9 @@ Release Notes
 
 1.7.2
 ----------------
+- Added 'read_alert_message' and 'handle_alert' to enhance the handling of alert messages.
```

Comment:
One last thing. Let's update this changelog message to reflect what actually changed. 

So something like ```added 'dismiss alert' and changed 'get alert message' to optionally leave alert open```. It doesn't have to be that exactly, but you get the idea.